components/linear/linear.app.mjs

```diff
@@ -11,5 +11,43 @@ export default {
         ...options,
       };
     },
+    async getOwnUserInfo() {
+      const {
+        avatarUrl, displayName,
+      } = await this.client().viewer;
+      return {
+        avatarUrl,
+        displayName,
+      };
+    },
+  },
+  propDefinitions: {
+    ...linearApp.propDefinitions,
+    createAs: {
+      type: "string",
+      label: "Create As User/App",
+      description: `Select who will appear as creating the issue on its history.
+        \\
+        See the [Linear docs](https://developers.linear.app/docs/oauth/oauth-actor-authorization) for more information.`,
```

Comment:
@GTFalcao is there a reason for the new line here? Can we condense to a single line?

Reply:
Changed to a single line.